gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/FieldValueTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.bigquery;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import com.google.api.services.bigquery.model.TableCell;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import org.junit.Test;
+
+import java.util.Map;
+
+public class FieldValueTest {
+
+  private static final TableCell BOOLEAN_FIELD = new TableCell().setV("false");
+  private static final Map<String, String> INTEGER_FIELD = ImmutableMap.of("v", "1");
```

Comment:
I am trying to test different types that can be passed to `fromPb`. First level field values (in a row) are passed as a `TableCell` object, while field values nested in a record are just an `ArrayMap` hence I am using `instance of Map` in `fromPb`.